src/button/disabled.scss

```diff
@@ -16,9 +16,29 @@
     }
   }
 
-  > .iui-icon {
+  > .iui-button-icon {
     @include themed {
       fill: t(iui-icons-color-actionable-disabled);
     }
   }
+
+  &:hover,
+  &:active {
```

Comment:
No reason to repeat all these rules. 

Ideally you won't even need these selectors but worst case, you can use `&, &:hover, &:active` if it doesn't work

Reply:
Deleted repeating code.